rpc/kate-rpc/src/lib.rs

```diff
@@ -157,6 +157,12 @@ macro_rules! internal_err {
 impl<Client, Block> Kate<Client, Block>
 where
 	Block: BlockT,
+	<Block as BlockT>::Header: ExtendedHeader<
```

Comment:
How did we end up needing this?

Reply:
It's go get the correct header and extension 
`let header_version = block_header.extension().get_header_version();`